website/registrations/utils.py

```diff
@@ -353,14 +353,16 @@ def _validate(self):
             parsed_contributor_list = [val.strip() for val in self.value.split(';')]
             for contrib in parsed_contributor_list:
                 match = self.contributor_regex.match(contrib.strip())
-                if match:
+                if match is not None:
```

Comment:
Is `if match` equal to `if match is not None`?

Reply:
12 lines above in `contributor_regex = re.compile(r'(?P<full_name>[\w\W]+)<(?P<email>.*?)>’)`, is it the `.*` that leads to matching empty email?